pkg/sqlcmd/sqlcmd_test.go

```diff
@@ -104,7 +104,7 @@ func TestSqlCmdQueryAndExit(t *testing.T) {
 		s.SetOutput(nil)
 		bytes, err := os.ReadFile(file.Name())
 		if assert.NoError(t, err, "os.ReadFile") {
-			assert.Equal(t, "Sqlcmd: Error: Syntax error at line 1."+SqlcmdEol, string(bytes), "Incorrect output from Run")
+			assert.Equal(t, "Sqlcmd: Error: Syntax error at line 1"+SqlcmdEol, string(bytes), "Incorrect output from Run")

```

Comment:
Yes. VS Code had actually suggested to avoid ending error strings with punctuation.
Also suggested as a good practice [here](https://github.com/golang/go/wiki/CodeReviewComments#error-strings)